client/src/components/menu-views/menu-item-dish.js

```diff
@@ -19,6 +19,23 @@ class MenuItemDish extends Component {
     };
   }
 
+  onDeleteClick(id) {
```

Comment:
Cool! Nice feature from another task https://trello.com/c/L7JOc9eO/40-39as-a-restaurant-owner-i-want-to-have-a-possibility-to-delete-dish
And it will be better to show alert with question "You try to delete dish. Are you sure?"

Reply:
I did not see that there is such task